x-pack/plugins/canvas/public/components/shape_preview/__examples__/__snapshots__/shape_preview.examples.storyshot

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Storyshots components/Shapes/ShapePreview arrow 1`] = `
+<div
+  className="canvasShapePreview"
+  dangerouslySetInnerHTML={
```

Comment:
It looks like `dangerouslySetInnerHTML` is often used in storybook storyshots, I'd like to learn from some link or a brief explanation. I suppose `html-react-parser` would be too convoluted, or it's just not important to bother with it, or there's some fundamental reason for not using this alternative

Reply:
@monfera That actually comes from `shape_preview`, not Storybook:

https://github.com/elastic/kibana/blob/master/x-pack/plugins/canvas/public/components/shape_preview/shape_preview.js#L31